commands/mute/tempmute.js

```diff
@@ -0,0 +1,115 @@
+const Discord = require('discord.js');
+var dateFormat = require('dateformat');
+const ms = require('ms');
+
+module.exports = {
+    name: 'tempmute',
+    description: 'Temporarily mute a user',
+
+    execute(msg, args, con) {
+        let reason = undefined;
+        let lengthOfTime = undefined;
+
+        if (!msg.member.roles.cache.some(
+        role => role.name === "Moderator" || role.name === "Admin")) {
+            return msg.reply("You must be a moderator or admin to use this command.");
+        } else {
+            if (!args.join(" ").match(/(\<@!?\d+\>)\s(\d+[yhwdms])\s(.+)$/)) {
+                return msg.reply("The command you sent isn't in a valid format.")
+            }
+
+            command_regex = /(\<@!?\d+\>)\s(\d+[yhwdms])\s(.+)$/;
```

Comment:
`command_regex` should probably be moved above this conditional so it can be changed to:
```javascript
if (!args.join(" ").match(command_regex)
```

Reply:
Done